Client/src/components/common/Table/index.css

```diff
@@ -0,0 +1,33 @@
+.Table {
+  margin: 50px auto;
+  border-spacing: 0px;
+  background: #fff;
+  box-shadow: 0 1px 0 0 rgba(22, 29, 37, 0.05);
```

Comment:
``` 1px ``` !!!!! i think it's invisible

Reply:
this code from the figma and it's work @knanahassouna1